nss-day-cohort-39/holiday-road-solar-space-sharks
# Ac save complete itinerary

Steps to test:
- make sure save trip button does not appear until you have selected all three required fields (park, food, and attraction)
- make sure that when the save button does appear and is clicked that the new trip object details render in the my trip section AND that the trip gets saved to the api and populates there.
